webapp-django/crashstats/signature/templates/signature/signature_report.html

```diff
@@ -135,6 +141,34 @@
         </div>
     </section>
 
+    <!-- Graph panel.
+        Displays a graph of crashes/ADU against build date
+    -->
+    <section class="panel" id="graph-panel"
+        data-source-url="{{ url('signature:signature_graph') }}"
+        data-jsonurl="{{ url('signature:signature_report') }}graphdata/"
```

Comment:
Just remembered why I did this: url('signature:signature_graph_data') didn't work, so I looked at the aggregations view - which is the other view where a second argument is passed through in addition to the request - and copied that pattern. Is there a better way to do it?